skyllh/i3/signal_generation.py

```diff
@@ -382,9 +382,9 @@ def signal_event_post_sampling_processing(
             n_sig = len(shg_src_sig_events)
 
             # Rotate the signal events to the source location.
-            (ra, dec) = rotate_spherical_vector(
-                shg_src_sig_events['true_ra'], shg_src_sig_events['true_dec'],
+            (ra, dec) = rotate_signal_events_on_sphere(
                 source.ra*np.ones(n_sig), source.dec*np.ones(n_sig),
```

Comment:
I think there shouldn't be a multiplication performed. Instead of multiplying with `np.ones(n_sig)`, one should use `np.full(n_sig, source.ra)` and `np.full(n_sig, source.dec)`